samples/ts/21.react-media-template/public/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
I can't leave comments on image files -- could you delete the unused logos and update this manifest? Not sure we need the robots.txt either.